lib/ansible/plugins/lookup/cloudformation_parse.py

```diff
@@ -0,0 +1,52 @@
+# (c) 2018, Steven Miller
+# (c) 2018 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
```

Comment:
Please add `requirements:` for `cfn_flip`

Example: https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/lookup/consul_kv.py#L20-L21